src/PathMatcher.php

```diff
@@ -0,0 +1,75 @@
+<?php declare (strict_types = 1);
+
+namespace OdbavTo\PresenterRoute;
+
+class PathMatcher
+{
+	/**
+	 * @var PathMatcher
+	 */
+	private static $instance;
+	
+	public static function getInstance(): PathMatcher
+	{
+		if (!self::$instance) {
+			self::$instance = new PathMatcher();
+		}
+		return self::$instance;
+	}
+	
+	private function __construct()
```

Comment:
You should respect ordering methods by their visibility public -> protected -> private.